src/Particles/babylon.particleSystem.ts

```diff
@@ -325,13 +325,33 @@
         private _scaledGravity = Vector3.Zero();
         private _currentRenderId = -1;
         private _alive: boolean;
+
         private _started = false;
         private _stopped = false;
         private _actualFrame = 0;
         private _scaledUpdateSpeed: number;
         private _vertexBufferSize = 11;
         private _isAnimationSheetEnabled: boolean;
 
+        // end of sheet animation
+
+        // Sub-emitters
+        /**
+         * this is the Sub-emitters templates that will be used to generate particle system when the particle dies, this property is used by the root particle system only.
+         */
+        public subEmitters: ParticleSystem[];
+        /**
+        * The current active Sub-systems, this property is used by the root particle system only.
+        */
+        public activeSubSystems: Array<ParticleSystem>;
+
+        private stockSubSystems: StringDictionary<Array<ParticleSystem>>;

```

Comment:
per our previous discussion we don;t need this :)

Reply:
Performance, If I remove it the FPS is dropping :(

Reply:
I do not get why (sorry :( ).
The stockSubSystem is used to store active ps, correct? And you use it to clone new PS instead of instancing new one.
How could this impact rendering perf?